website/settings/defaults.py

```diff
@@ -283,3 +284,4 @@ def parent_dir(path):
 
 ENABLE_ARCHIVER = True
 ###########################
+REGISTRATION_APPROVAL_PERIOD = datetime.timedelta(days=2)
```

Comment:
We don't, Sam and I probably added those independently of one another. I'll consolidate them.